src/exceptions.jl

```diff
@@ -1,16 +1,81 @@
 # The exceptions defined here mostly mirror the exceptions supported by Ray for Python:
 # https://github.com/ray-project/ray/blob/ray-2.5.1/python/ray/exceptions.py
 
+struct ObjectContext
+    object_ref_hex::String
+    owner_address::ray_jll.Address
+    call_site::String
+end
+
+function ObjectContext(obj_ref::ObjectRef)
+    return ObjectContext(hex_identifier(obj_ref), get_owner_address(obj_ref), "")
```

Comment:
is this really not covered by tests or is it an inlining failure?

Reply:
It's not covered by tests. This is the only part of the this code that requires Ray.jl to be connected to the raylet. As the majority of the code doesn't require it I opted to test this source file without being connected to the raylet. As a follow up I think I'd make a convert method in `object_ref.jl` and test this portion that way.

I'll make an issue

Reply:
https://github.com/beacon-biosignals/Ray.jl/issues/231